libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -86,15 +87,20 @@ - (void)setUp {
     // Delete the content of the global library directory
     NSString *globalLibraryDirectory = [[SFDirectoryManager sharedManager] directoryForUser:nil type:NSLibraryDirectory components:nil];
     [[[NSFileManager alloc] init] removeItemAtPath:globalLibraryDirectory error:nil];
-
     // Set the oauth client ID after deleting the content of the global library directory
     // to ensure the SFUserAccountManager sharedInstance loads from an empty directory
     self.uam = [SFUserAccountManager sharedInstance];
-    self.uam.oauthClientId = @"fakeClientIdForTesting";
+    [SFAuthenticationManager sharedManager].oauthClientId = @"fakeClientIdForTesting";
```

Comment:
And by inline-set it, I mean just use the literal value in the call to `[SFOAuthCredentials initWithIdentifier:clientId:encrypted:]`.  We don't need to bootstrap `SFAuthenticationManager` in this case.